matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/model/RoomSummaryEntity.kt

```diff
@@ -235,6 +235,8 @@ internal open class RoomSummaryEntity(
             if (value != field) field = value
         }
 
+    var directParentNames: RealmList<String> = RealmList()
```

Comment:
Not a blocker, but If there is no `set` fun, this could be moved to the constructor.